src/hotspot/os/aix/os_aix.cpp

```diff
@@ -295,6 +296,8 @@ static bool my_disclaim64(char* addr, size_t size) {
 
   for (unsigned int i = 0; i < numFullDisclaimsNeeded; i ++) {
     if (::disclaim(p, maxDisclaimSize, DISCLAIM_ZEROMEM) != 0) {
+      ErrnoPreserver ep;
+      log_trace(os,map)("Cannot disclaim %p - %p (errno %d)\n", p, p + maxDisclaimSize, errno);
       trcVerbose("Cannot disclaim %p - %p (errno %d)\n", p, p + maxDisclaimSize, errno);
```

Comment:
I don't think you need actual unified logging and the crude `trcVerbose`, but perhaps that would be a separate cleanup. Though the duplication is pretty awful.

Reply:
Well as the whole point of this PR is to add UL in this area, I suggest leaving the duplication and cleanup/remove trcVerbose seprately.

Reply:
Please remove trcVerbose. 

I'm the original author of it (stems from SAPs AIX port originally, where output would flow into a different logging system).